code/modules/transport/pods/secondary_system.dm

```diff
@@ -585,6 +595,31 @@
 			src.ship.myhud.update_states()
 		boutput(usr, SPAN_NOTICE("Thrusters will now provide ship movement to the [src.turn_dir]."))
 
+/obj/item/shipcomponent/secondary_system/thrusters/afterburner
+	name = "Afterburner"
+	desc = "An engine augment that enhances the burning of plasma, increasing maximum velocity for a short duration."
+	hud_state = "lat_thrusters_right"
+	f_active = TRUE
+	power_used = 50
+
+	use_thrusters(mob/user)
+		if (ON_COOLDOWN(src, "thruster_movement", 20 SECONDS))
+			boutput(user, "[src.ship.ship_message("Afterburner is recharging! [round(GET_COOLDOWN(src, "thruster_movement") / 10, 0.1)] seconds left.")]")
+			return
+
+		// spawn to allow button clunk sound to play right away
+		SPAWN(0)
+			boutput(user, "[src.ship.ship_message("Afterburner is now active!")]")
```

Comment:
Ideally this should have some kind of visual or audio feedback, maybe just a fwoosh sound for people in the cabin and a little puff of flame or something to show other pod pilots that it's been used.